APMrover2/Log.cpp

```diff
@@ -198,9 +206,12 @@ void Rover::Log_Write_Steering()
 {
     struct log_Steering pkt = {
         LOG_PACKET_HEADER_INIT(LOG_STEERING_MSG),
-        time_us        : AP_HAL::micros64(),
-        demanded_accel : lateral_acceleration,
-        achieved_accel : ahrs.groundspeed() * ins.get_gyro().z,
+time_us        :
```

Comment:
and here and in every designated initializer.   I wonder why we are using gnu90 designated initializer rather than C99 one.